flutter_local_notifications/README.md

```diff
@@ -307,6 +307,7 @@ await flutterLocalNotificationsPlugin.initialize(initializationSettings,
 ```
 
 Initialisation can be done in the `main` function of your application or can be done within the first page shown in your app. Developers can refer to the example app that has code for the initialising within the `main` function. The code above has been simplified for explaining the concepts. Here we have specified the default icon to use for notifications on Android (refer to the *Android setup* section) and designated the function (`selectNotification`) that should fire when a notification has been tapped on via the `onSelectNotification` callback. Specifying this callback is entirely optional but here it will trigger navigation to another page and display the payload associated with the notification.
+Note that all settings are nullable, because we don't want to force developers so specify settings for platforms they don't target. You will get a runtime ArgumentError Exception if you forgot to pass the settings for the platform you target.
```

Comment:
Given how you wrote this note on its own line, did you intend for this be its own paragraph? If so then you're missing a line break to make this happen. Without the line break this sentence actually continues to be part of the same paragraph so can make it more confusing to match what's written "in code" with how it should appear